In [3]:
import pandas as pd

In [4]:
df = pd.read_csv('data/tianchi_fresh_comp_train_user.csv')

In [5]:
df.describe()

user_id       item_id  behavior_type  item_category
count  2.329103e+07  2.329103e+07   2.329103e+07   2.329103e+07
mean   7.006868e+07  2.023214e+08   1.106268e+00   6.835397e+03
std    4.569072e+07  1.167440e+08   4.599087e-01   3.812873e+03
min    4.920000e+02  3.700000e+01   1.000000e+00   2.000000e+00
25%    3.019541e+07  1.014417e+08   1.000000e+00   3.690000e+03
50%    5.626942e+07  2.022430e+08   1.000000e+00   6.054000e+03
75%    1.166482e+08  3.035325e+08   1.000000e+00   1.027100e+04
max    1.424430e+08  4.045625e+08   4.000000e+00   1.408000e+04

In [6]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 23291027 entries, 0 to 23291026
Data columns (total 6 columns):
 #   Column         Dtype 
---  ------         ----- 
 0   user_id        int64 
 1   item_id        int64 
 2   behavior_type  int64 
 3   user_geohash   object
 4   item_category  int64 
 5   time           object
dtypes: int64(4), object(2)
memory usage: 1.0+ GB


In [7]:
items = pd.read_csv('data/tianchi_fresh_comp_train_item.csv')
items.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 620918 entries, 0 to 620917
Data columns (total 3 columns):
 #   Column         Non-Null Count   Dtype 
---  ------         --------------   ----- 
 0   item_id        620918 non-null  int64 
 1   item_geohash   203410 non-null  object
 2   item_category  620918 non-null  int64 
dtypes: int64(2), object(1)
memory usage: 14.2+ MB


```tianchi_fresh_comp_train_user.csv```  

Item | Value 
- | -
Data Size | 23291027
Columns | 0-user_id-int64, 1-item_id-int64, 2-behavior_type-int64, 3-user_geohash-object, 4-item_category-int64, 5-time-object
Memory Usage | 1.0+ GB

```tianchi_fresh_comp_train_item.csv```  

Item | Value
- | -
Data Size | 620918
Columns | 0-item_id-int64, 1-item_geohash-object, 2-item_category-int64
Memory Usage | 14.2+ MB